docs/experiment/ppl/cmd/head.rst

```diff
@@ -16,10 +16,8 @@ Description
 
 Syntax
 ============
-head [keeplast = (true | false)] [while "("<boolean-expression>")"] [N]
```

Comment:
I'm thinking should we document this breaking change here or elsewhere

Reply:
Could we 
1. add the breaking change description in this PR.
2. add the breaking change label for the PR.
Then, during the release stage, 
we can pull the breaking change and describe it in the release notes?